wikidatarefisland/pipes/PipeInterface.py

```diff
@@ -0,0 +1,7 @@
+from abc import ABC, abstractmethod
+
+
+class PipeInterface(ABC):
+    @abstractmethod
+    def flow(self, data: dict) -> dict:
```

Comment:
Does the explicit return type mean that we have to return `dict` from the `flow` method?

Reply:
I have the same question too. For example my pipe take a row and produces multiple rows and I don't see a way around it :(

Reply:
It does! I think I should change it then.
What should it be changed to? 
- No type hinting at all?
- String (e.g. the pipe returns JSON/JSONl)
- Iterable

I think Iterable sounds like it makes most sense now

Reply:
+1 on iterable

Reply:
What about me having to return `None` from pipe 3?

Reply:
> What about me having to return `None` from pipe 3?

Good point, what about [] instead of None. That's still empty and iterable.